tests/integration/sms/contacts_interaction_test.js

```diff
@@ -0,0 +1,115 @@
+/* global require, marionette, setup, suite, test */
+'use strict';
+
+var assert = require('chai').assert;
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var FromApp = require(appRoot + '/shared/test/integration/require_from_app');
+
+var Messages = FromApp.require('sms', 'lib/messages');
+var InboxView = FromApp.require('sms', 'lib/views/inbox/view');
+var ThreadGenerator = FromApp.require('sms', 'generators/thread');
+var Storage = FromApp.require('sms', 'lib/storage.js');
+
+var Contacts = FromApp.require('contacts', 'lib/contacts');
+var ContactsData = FromApp.require('contacts', 'lib/contacts_data');
+
+marionette('Contacts', function() {
+  var messagesApp;
+
+  var client = marionette.client({
+    prefs: {
+      'focusmanager.testmode': true
+    }
+  });
+
+  var thread;
```

Comment:
since this file is out of scope of my responsibility, here just feedback you can ignore :)

We usually join uninitialized variables together ```var a, b, c;```, but never do this for initialized variables.